hicommonwealth/commonwealth
# Chain-Events chain -> community semantics

## TODO:
- [ ] Ensure `CE_default.dump` is updated to the latest migration for CI (new migration must be applied to `CE_default.dump`)  -> to do this: get a fresh CE dump then run 
```
pg_dump postgresql://commonwealth:edgeware@localhost/commonwealth_chain_events --exclude-table-data="public.\"ChainEntities\"" --exclude-table-data="public.\"ChainEvents\"" --verbose --no-privileges --
no-owner -f CE_default.dump
```

## Link to Issue
Closes: #3881 

## Description of Changes
WIP

## Test Plan
- EVM integration tests
- Deploy to chain-events-staging and observe the `subscriber` (1, 2,  and 3) logs to ensure no errors arise. You can also leave it running for an extended period of time and then check the database to ensure new events are being created.
- Load proposals in `edgeware`, `dydx` and `cosmos`

## Deployment Plan
1. Stop the CE subscribers + consumer
2. Deploy this PR CE first (migration in CE must finish before deploying to CW)
3. Deploy this PR to CW once CE deployment is released
4. Merge TBD PR
5. Deploy TBD PR to CE to remove the legacy columns and API fetching from CE

## Other Considerations

-
